### DISCUSSION TITLE
Rss feed

### DISCUSSION THREAD

**Discussion by TeckboyAj** [stringer-rss/stringer#1074] [Q&A]
I want to ask is it possible that the content which we have published on the fressrss, we can export all the post.in form of xml file. Is it possible?

**Comment by mockdeep** [comment]
Hey @TeckboyAj I don't think this is the place for your question. It sounds like you're trying to find a way to export items you have published, but Stringer is just an RSS reader.

**Reply by TeckboyAj** [reply]
Okay
